rskj-core/src/test/java/co/rsk/config/ConfigUtils.java

```diff
@@ -25,7 +25,7 @@ public static MiningConfig getDefaultMiningConfig() {
         return new MiningConfig(
                 new RskAddress(coinbaseAddress),
                 0.0,
-                0.0,
+                1.0,
```

Comment:
why do we need this change?

Reply:
as I understand, tests don't need this change as of now. They would pass even without this change, wouldn't they?